<?xml version="1.0" encoding="UTF-8"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="UMIUSI">
  <xacro:macro name="imu">
    <ros2_control name="imu" type="sensor">
      <hardware>
        <plugin>sinsei_umiusi_control/hardware/Imu</plugin>
      </hardware>
      <sensor name="imu">
        <state_interface name="orientation_raw.x" />
        <state_interface name="orientation_raw.y" />
        <state_interface name="orientation_raw.z" />
        <state_interface name="velocity_raw.x" />
        <state_interface name="velocity_raw.y" />
        <state_interface name="velocity_raw.z" />
        <state_interface name="temperature" />
      </sensor>
    </ros2_control>
  </xacro:macro>
</robot>